airbyte-scheduler/scheduler-persistence/src/main/java/io/airbyte/scheduler/persistence/DefaultJobPersistence.java

```diff
@@ -340,17 +341,70 @@ private Optional<Job> getJobOptional(final DSLContext ctx, final long jobId) {
     return getJobFromResult(ctx.fetch(BASE_JOB_SELECT_AND_JOIN + "WHERE jobs.id = ?", jobId));
   }
 
+  @Override
+  public Long getJobCount(final Set<ConfigType> configTypes, final String connectionId) throws IOException {
+    return jobDatabase.query(ctx -> ctx.selectCount().from(JOBS)
+        .where(JOBS.CONFIG_TYPE.cast(String.class).in(Sqls.toSqlNames(configTypes)))
```

Comment:
We should be able to without the cast:
```JOBS.CONFIG_TYPE.in(configTypes)```

Reply:
You're right that we don't need the `cast` anymore now that we are doing things this way with jooq. However, we do still need the `Sqls.toSqlNames(configTypes)` call in the value, or else `getSQL()` produces `null`s for the values.

Reply:
I've pushed up the change to remove those unnecessary `cast()` calls